build/openshift/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG GOLANG_CONTAINER=registry.access.redhat.com/ubi8/go-toolset:latest
+ARG GOLANG_CONTAINER=golang:latest
```

Comment:
I would double check it with @Rulox

Reply:
Yeah, this is fine.

The main issue is that `ubi-go` image latest is Go 1.12. RedHat keeps those images outdated so there's nothing we can do. We need 1.13 for the new k8s libraries. I made some tests, both images work to compile the IC Binary, because the GOOS is `linux` in both.

The alternative was to create our own ubi based image, install Go manually and use that.

I tested with golang:latest and it works for ubi. Think of this like the `alpine` image. We use golang:latest as well there.